packages/graphql-codegen-cli/src/codegen.ts

```diff
@@ -65,87 +68,211 @@ async function mergeSchemas(schemas: GraphQLSchema[]): Promise<GraphQLSchema> {
 }
 
 export async function executeCodegen(config: Types.Config): Promise<FileOutput[]> {
-  const result = [];
+  const result: FileOutput[] = [];
+  const commonListrOptions = {
+    exitOnError: true
```

Comment:
It doesn't mean `process.exit()`, it just stops other tasks when something failed.